### PR TITLE
chore: prepare Tokio 1.0 release

### PR DESCRIPTION
**Pull request by carllerche**

### CODE REVIEW

File: tokio/CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,53 @@
...
+- fs: `poll_*` fns to `DirEntry` (#3308).
+- io: `poll_*` fns to `io::Lines`, `io::Split` (#3308).
+- io: `_mut` method variants to `AsyncFd` (#3304).
+- net: `poll_*` fns to `UnixDatagram` (#3223).
+- net: `UnixStream` readiness and non-blocking ops (#3246).
+- sync: `UnboundedReceiver::blocking_recv()` (#3262).
+- sync: `watch::Sender::borrow()` (#3269).
+- sync: `Semaphore::close()` (#3065).
+- sync: `poll_*` fns to `mpsc::Receiver`, `mpsc::UnboundedReceiver` (#3308).
+- time: `poll)*` fns to `time::Interval` (#3316).
```
Comment (taiki-e) [inline tokio/CHANGELOG.md:36]:
```suggestion
- time: `poll_*` fns to `time::Interval` (#3316).
```

File: tokio-macros/CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,7 @@
+# 0.4.0 (December 23, 2020)
```
Comment (paolobarbolini) [inline tokio-macros/CHANGELOG.md:1]: Shouldn't this be 1.0.0?

File: tokio/CHANGELOG.md (hunk 1 above)
Comment (Darksonn) [inline tokio/CHANGELOG.md:36]:
```suggestion
- sync: `poll_recv` fns to `mpsc::Receiver`, `mpsc::UnboundedReceiver` (#3308).
- time: `poll_tick` fn to `time::Interval` (#3316).
```

File: tokio-macros/LICENSE [hunk 3]
```diff
@@ -1,4 +1,4 @@
-Copyright (c) 2019 Tokio Contributors
+Copyright (c) 2020 Tokio Contributors
```
Comment (LucioFranco) [inline tokio-macros/LICENSE:1]: should just change this to 2021 😆

**Review by LucioFranco — APPROVED** [review]

File: tokio-macros/LICENSE (hunk 3 above)
Comment (carllerche) [reply tokio-macros/LICENSE:1]: next year!

**Comment by taiki-e** [comment]
Some examples still use 0.3.
https://github.com/tokio-rs/tokio/blob/c6a1b8eef9f36566e903e8b8d9d795d48c23247b/tokio/src/lib.rs#L60
https://github.com/tokio-rs/tokio/blob/c6a1b8eef9f36566e903e8b8d9d795d48c23247b/tokio/src/lib.rs#L75
https://github.com/tokio-rs/tokio/blob/c6a1b8eef9f36566e903e8b8d9d795d48c23247b/tokio/src/lib.rs#L91

**Pull request merged**